pandas/plotting/_matplotlib/hist.py

```diff
@@ -246,8 +246,7 @@ def _grouped_plot(
     if figsize == "default":
         # allowed to specify mpl default with 'default'
         raise ValueError(
-            "figsize='default' is no longer supported. "
-            "Specify figure size by tuple instead"
+            "figsize='default' is no longer supported.Specify figure size by tuple instead"
```

Comment:
Where is the typo here? Looks like a space was just removed and line made too long